lib/xgu/xgux.h

```diff
@@ -13,11 +13,11 @@ void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int co
 }
 
 inline
-void xgux_set_clear_rect(uint32_t left, uint32_t right,
-                         uint32_t top, uint32_t bottom) {
+void xgux_set_clear_rect(unsigned int x, unsigned int y,
+                         unsigned int width, unsigned int height) {
     uint32_t *p = pb_begin();
-    p = xgu_set_clear_rect_horizontal(p, left, right);
-    p = xgu_set_clear_rect_vertical(p, top, bottom);
+    p = xgu_set_clear_rect_horizontal(p, x, x+width);
```

Comment:
I believe the old code implied `x+width-1`, although I'm not entirely sure myself.

Reply:
It's been a while and I'm guessing you've done some testing. Do you reckon I should make this suggested change?